decksite/main.py

```diff
@@ -94,6 +94,10 @@ def achievements():
     view = Achievements(auth.mtgo_username())
     return view.page()
 
+@APP.route('/person/achievements/')
```

Comment:
@adambiltcliffe Also try not to break old URLs.  
If you change any other URLs in the future, please add a redirect.  (It's bad for SEO, old links, and people's bookmarks)